tensorflow/contrib/ios_examples/camera/CameraExampleViewController.mm

```diff
@@ -22,6 +22,27 @@
 
 #include "tensorflow_utils.h"
 
+// If you have your own model, modify this to the file name, and make sure
+// you've added the file to your app resources too.
+static NSString* model_file_name = @"tensorflow_inception_graph";
+static NSString* model_file_type = @"pb";
+// This controls whether we'll be loading a plain GraphDef proto, or a
+// file created by the convert_graphdef_memmapped_format utility that wraps a
+// GraphDef and parameter file that can be mapped into memory from file to
+// reduce overall memory usage.
+const bool model_uses_memory_mapping = false;
```

Comment:
I believe constant variables have internal linkage (like file-level static) by default[1], so it's prefered to avoid prefixing them with static. The NSStrings are defined that way because they're not const, mostly because they're based on old-school C constructs through ObjectiveC, so I explicitly make them static to avoid possible linker conflicts. 

[1] http://stackoverflow.com/questions/6173872/why-is-multiple-definition-of-a-const-global-variable-allowed-in-c-and-not-in